app/controllers/users_controller.rb

```diff
@@ -1740,6 +1740,68 @@ def bookmarks
     end
   end
 
+  USER_MENU_BOOKMARKS_LIST_LIMIT = 20
+  def user_menu_bookmarks
+    user = fetch_user_from_params
+    guardian.ensure_can_see_notifications!(user)
+    user_guardian = Guardian.new(user)
+
+    if params[:limit]
```

Comment:
Is there a reason we need to support a custom limit for this end point right now?

Reply:
It's not actually used, but I couldn't figure out a way to test the limit and per_page stuff without a limit param and without fabricating 20+ objects in the tests which would make the test slow. Any ideas?

Reply:
Can you stub the constant in the test instead? We have a `stub_const` helper for that.

Reply:
done!